taxi/templatetags/query_transform.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def query_transform(request, **kwargs):
+    updated = request.GET.copy()
+    for key, value in kwargs.items():
+        if value is not None:
+            updated[key] = value
+        else:
+            updated.pop(key, 0)
+
+    return updated. urlencode()
```

Comment:
Remove the space after `.` here
```suggestion
    return updated.urlencode()
```